client/reader/stream/reader-sidebar-followed-sites/index.jsx

```diff
@@ -41,35 +35,11 @@ export class ReaderSidebarFollowedSites extends Component {
 		sitesPerPage: PropTypes.number,
 	};
 
-	handleReaderSidebarFollowedSitesClicked = () => {
-		recordAction( 'clicked_reader_sidebar_followed_sites' );
-		recordGaEvent( 'Clicked Reader Sidebar Followed Sites' );
-		this.props.recordReaderTracksEvent( 'calypso_reader_sidebar_followed_sites_clicked' );
```

Comment:
Seems like `recordReaderTracksEvent` isn't used anymore.

Reply:
Thanks! Removed in https://github.com/Automattic/wp-calypso/pull/70667/commits/55fef8f864a925f995a599715a8fe888746bdec8